Difi.SikkerDigitalPost.Klient/Difi.SikkerDigitalPost.Klient.csproj

```diff
@@ -60,11 +60,19 @@
       <HintPath>..\packages\difi-felles-utility-dotnet.0.8.0.27738\lib\net45\Difi.Felles.Utility.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+      <HintPath>..\packages\Newtonsoft.Json.6.0.4\lib\net45\Newtonsoft.Json.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
     <Reference Include="System" />
     <Reference Include="System.Configuration" />
     <Reference Include="System.Core" />
     <Reference Include="System.IO.Compression" />
     <Reference Include="System.Net.Http" />
+    <Reference Include="System.Net.Http.Formatting, Version=5.2.3.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
Dette er litt interessant. Jeg har alltid bare gitt blaffen i endringer i .csproj fordi den er autogenerert. Ser at den er i navnerommet `System.Net.Http` som er det samme som for `HttpClientFactory`, så regner med at det bare er noe som trengs i den. Den får nok bare denne fra `.NET`-rammeverket og blir derfor ikke en pakkeavhengighet.